src/libraries/Common/tests/TestUtilities.Unicode/System/Text/Unicode/ParsedUnicodeData.cs

```diff
@@ -103,14 +103,14 @@ public ParsedUnicodeData()
 
             Dictionary<int, BidiClass> dict = new Dictionary<int, BidiClass>();
 
-            string thisLine;
+            string? thisLine;
             while ((thisLine = reader.ReadLine()) != null)
             {
-                if (PropsFileEntry.TryParseLine(thisLine, out PropsFileEntry value))
+                if (PropsFileEntry.TryParseLine(thisLine, out PropsFileEntry? value))
```

Comment:
Rather than using `!` to suppress nullable warnings on value below, can we add `[NotNullWhen(true)]` to the TryParseLine out parameter?

Reply:
Will do that.

Reply:
Seems like you should be able to remove the `!` now right?